Assets/MixedRealityToolkit/Sharing/README.md

```diff
@@ -166,7 +166,7 @@ Draws the default Sharing Stage Inspector and adds the SyncRoot Hierarchy view s
 
 #### [SDK](Scripts/SDK)
 ---
-Contains scripts compiled from the native [MixedRealityToolkit\Sharing](https://github.com/Microsoft/MixedRealityToolkit/tree/Dev_Working_Branch/Sharing) repository and using the SWIG tool to generate different language bindings.
+Contains scripts compiled from the native [MixedRealityToolkit\Sharing](https://github.com/Microsoft/MixedRealityToolkit/tree/master/Sharing) repository and using the SWIG tool to generate different language bindings.
```

Comment:
Could we shorten this to just `/Sharing`? like we did above?

Reply:
This one links to a different repo, the non-Unity one

Reply:
Oh I see. I wasn't aware we had a Dev_Working_Branch there.

Reply:
There isn't a dev_working_branch on the base MRTK, hence why I had to revert :D